src/api/api.ts

```diff
@@ -0,0 +1,14 @@
+const getData = async (url: string) => {
+  const response = await fetch(url);
+  const json = await response.json();
+
+  return json;
+};
+
+export const getMovie = async (value: string) => {
+  const API_KEY = '45abeff6';
```

Comment:
you better store this variable in .env file since it's a confidential data (optional comment)
check this https://create-react-app.dev/docs/adding-custom-environment-variables/